pallets/parachain-system/src/lib.rs

```diff
@@ -58,7 +58,7 @@ use sp_std::{cmp, collections::btree_map::BTreeMap, prelude::*};
 use xcm::latest::XcmHash;
 
 mod migration;
-mod relay_state_snapshot;
+pub mod relay_state_snapshot;
```

Comment:
Either:
  * re-export the necessary types for this module and leave it private
  * add module docs to `relay_state_snapshot` and move this declaration alongside the other public modules.